Formula/zeek.rb

```diff
@@ -26,42 +26,46 @@ class Zeek < Formula
 
   depends_on "bison" => :build
   depends_on "cmake" => :build
+  depends_on "flex" => :build
   depends_on "swig" => :build
-  depends_on "caf"
-  depends_on "geoip"
+  depends_on "c-ares"
```

Comment:
- system `flex` is too old
- upstream no longer supports external `caf` - https://github.com/Homebrew/homebrew-core/pull/111218#issuecomment-1287772833
- `geoip` was replaced by `libmaxminddb`
- looks like we can use Homebrew's `c-ares`.